src/2_RTB/AnalisiDeiRequisiti/content.typ

```diff
@@ -345,45 +295,35 @@ Relativamente all'utilizzo della dashboard, viene definito un unico attore con a
 - *Postcondizioni*: l'attore visualizza un pannello contenente dati relativi al dominio urbanistico.
 - *Scenario Principale*:
   + l'attore accede al sistema;
-  + l'attore seleziona la visualizzazione del dominio urbanistico;
-  + il sistema elabora i dati e renderizza il pannello relativo.
+  + l'attore seleziona la visualizzazione del dominio urbanistico.
 - *Specializzazioni*: [UC3.2],[UC3.3],[UC3.4],[UC3.5],[UC3.6].
-- *Inclusioni*: nessuna.
 - *Estensioni*: [UC9].
 
 #figure(
   image("assets/UML/UC3.1_Visualizzazione-pannello-dati-urbanistici.png",width:100%),
   caption: [UC3.1 Visualizzazione pannello dati urbanistici]
 )
 
-=== UC3.2: Visualizzazione grafico a mappa disponibilità dei parcheggi
+=== UC3.2: Visualizzazione grafico a mappa disponibilità e occupazione parcheggi
 - *Attore Principale*: amministratore pubblico.
 - *Precondizioni*: l'attore ha selezionato la visualizzazione relativa al dominio dei dati urbanistici.
-- *Postcondizioni*: l'attore visualizza un grafico a mappa atto ad indicare le disponibilità nei parcheggi. 
+- *Postcondizioni*: l'attore visualizza un grafico a mappa atto ad indicare la disponibilità di parcheggi nella zona visualizzata e la loro occupazione, in termini di parcheggi liberi.
 - *Scenario Principale*:
   + l'attore accede al sistema;
-  + l'attore seleziona la visualizzazione del dominio urbanistico;
-  + il sistema elabora i dati e renderizza il pannello relativo.
-- *Specializzazioni*: nessuna. 
-- *Inclusioni*: nessuna.
-- *Estensioni*: nessuna.
+  + l'attore seleziona la visualizzazione del dominio urbanistico.
 
 #figure(
   image("assets/UML/UC3.2_Visualizzazione-grafico-a-mappa-disponibilità-dei-parcheggi.png",width:100%),
-  caption: [UC3.2 Visualizzazione grafico a mappa disponibilità dei parcheggi]
+  caption: [UC3.2 Visualizzazione grafico a mappa disponibilità e occupazione dei parcheggi]
```

Comment:
mi sembre ridondante ma si puo tenere

Reply:
Mi è stato specificamente richiesto maggiore dettaglio

Reply:
si ma disponibilità e occupazione sono esattamente la stessa cosa. Il dettaglio forse andava messo sul unità della misuarazione

Reply:
Disponibilità l'ho inteso come: esistenza di un parcheggio in una determinata zona

Reply:
perfavore sistema questa

Reply:
devi esplicitare il fatto che la diponibilità viene segnalata in termini di singoli parcheggi liberi

Reply:
è già indicato nelle post-condizioni